libs/domains/clusters/feature/src/lib/cluster-card/cluster-card.tsx

```diff
@@ -89,15 +82,9 @@ export interface ClusterCardProps {
 export function ClusterCard({ cluster, clusterDeploymentStatus }: ClusterCardProps) {
   useClusterRunningStatusSocket({ organizationId: cluster.organization.id, clusterId: cluster.id })
 
-  const isClusterOverviewEnabled = useFeatureFlagVariantKey('cluster-running-status')
```

Comment:
@RemiBonnet  
For this feature it's alright with me to remove the feature flag in one go like this, but for more complex and/or sensitive features, we could maybe keep it in the code for a small amount of time (let's say about a week), enable it for everyone from Posthog, and then remove it from the code as part of a second PR. 
What do you think? In my opinion it is safer that way.

Reply:
Yes I totally agree should be definitively more safe 💯